src/client/dfuse/pil4dfs/int_dfs.c

```diff
@@ -140,8 +140,6 @@ static _Atomic uint32_t        daos_init_cnt;
  * true.
  */
 static bool             report;
-/* always load libpil4dfs related env variables in exec() */
-static bool             enforce_exec_env;
```

Comment:
can you please explain why you are removing this now?

Reply:
We provide two modes with this PR, whitelist mode and interception-on all time, when libpil4dfs is loaded. Both need that children processes inherit parent process's pil4dfs related env variables. It makes more sense to always set "enforce_exec_env" on. If we expose this variable with env, the state is not well-defined if user sets "enforce_exec_env" off.
I do not see any obvious drawback to append pil4dfs related env variables in children processes if parent process preloads libpil4dfs.so already.